src/sagemaker/parameter.py

```diff
@@ -60,9 +61,16 @@ def as_tuning_range(self, name):
         Returns:
             dict[str, str]: A dictionary that contains the name and values of the hyperparameter.
         """
-        return {'Name': name,
-                'MinValue': to_str(self.min_value),
-                'MaxValue': to_str(self.max_value)}
+        tuning_range = {
+            'Name': name,
+            'MinValue': to_str(self.min_value),
+            'MaxValue': to_str(self.max_value)
+        }
+
+        if self.scaling_type is not None:
+            tuning_range['ScalingType'] = self.scaling_type
```

Comment:
Are there valid values for this? If so those should be added to the DocStrings and validated in here.